### DISCUSSION TITLE
Arrow suggestion

### DISCUSSION THREAD

**Discussion by kk-bg** [WindowTop/WindowTop-App#265] [Ideas]
Can you add an option to make the arrow invisible or not appear at all so we don't have to see the arrow to use the toolbar?
<img width="822" alt="WindowTop arrow suggestion (2022-10-07_16-52-00)" src="https://user-images.githubusercontent.com/115268743/194595915-ccba3f93-d9a9-48a8-a6a4-f3829aeae222.png">

**Comment by gileli121** [comment]
Thanks for the feedback.
What about the "Integrate with the minimize button" option?
Did you try it? If so, why do you prefer an invisible arrow?
Don't you think you will have difficulty locating the invisible arrow?

**Reply by kk-bg** [reply]
> Thanks for the feedback. What about the "Integrate with the minimize button" option? Did you try it? If so, why do you prefer an invisible arrow? Don't you think you will have difficulty locating the invisible arrow?

I need to enable the toolbar (which includes the arrow) in order to enable the "Integrate with the minimize button" option. It doesn't let me just choose one.
